Day3/src/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./style.css";
+import image from "./assets/logo.png";
+
+const Navbar = () => {
+  return (
+    <div className='navbar'>
+      <div className='image__container'>
+        <img src={image} />
+      </div>
+      <ul>
```

Comment:
So navbar required logo, searchbar and profile picture. can you modify with the requirements?

Reply:
You're saying to remove the unordered list?

Reply:
Got it.